.yarnrc

```diff
@@ -1,5 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
-# yarn lockfile v1
-
-
-yarn-path ".yarn/releases/yarn-1.19.1.js"
```

Comment:
Isn't this used to make sure everyone is running the same version of `yarn`? I know we recommend using volta now, but if there's no harm in having this too as the default way yarn handles version locking, I'd probably have both settings (just set to the same version).

Reply:
With Volta we're pinning to yarn 1.22.19 so I don't think this file would be used anyway

Reply:
Yeh, but you can pin the yarn version to `1.22.19` by downloading the v1.22.19 release file and including that instead. Then yarn-path should also point to that file (as a backup for anyone now using Volta)

v1.22.19 file [is available here](https://github.com/yarnpkg/yarn/releases/download/v1.22.19/yarn-1.22.19.js)